demo/launch/pipeline_testbench.launch.py

```diff
@@ -40,6 +40,7 @@ def launch_setup(context, *args, **kwargs):
         MoveItConfigsBuilder("moveit_resources_panda")
         .robot_description(file_path="config/panda.urdf.xacro")
         .planning_pipelines(pipelines=["ompl", "stomp"])
+        # .planning_pipelines(pipelines=["drake"])
```

Comment:
is this being used at all? Or is it overridden by the YAML file above?

Reply:
@sjahr I left the other pipelines as is for the final demo, for now yes it just uses the drake pipeline for testing and dev.